T0806 mg fix nested transaction error (#295)
memgraph/docs: docs/reference-guide/streams/overview.md

batch interval duration|Maximum wait time in milliseconds for consuming messages before calling the transform procedure|int|9999|100
batch size|Maximum number of messages to wait for before calling the transform procedure|int|99|1000
bootstrap servers|Comma-separated list of bootstrap servers|string|"localhost:9092"|/

> configs|String key-value pairs of configuration options for the Kafka consumer|map with string key-value pairs|{"sasl.username": "michael.scott"}|/
> credentials|String key-value pairs of configuration options for the Kafka consumer, but their value aren't shown in the Kafka specific stream information|map with string key-value pairs|{"sasl.password": "password"}|/
> 
> :::warning
> The credentials are stored on the disk without any encryption, that means everybody who has access to the data folder of Memgraph is able to get the credentials.

Reviewer: ```suggestion
configs|String key-value pairs of configuration options for the Kafka consumer|map with string key-value pairs|{"sasl.username": "michael.scott"}|/
credentials|String key-value pairs of configuration options for the Kafka consumer, but their value isn't shown in the Kafka specific stream information|map with string key-value pairs|{"sasl.password": "password"}|/

:::warning
The credentials are stored on the disk without any encryption, which means everybody who has access to the data folder of Memgraph is able to get the credentials.
```
